hpat/datatypes/hpat_pandas_series_functions.py

```diff
@@ -356,6 +357,65 @@ def hpat_pandas_series_append_impl(self, to_append):
     return hpat_pandas_series_append_impl
 
 
+@overload_method(SeriesType, 'isna')
+def hpat_pandas_series_isna(self):
+    """
+    Pandas Series method :meth:`pandas.Series.append` implementation.
+
+    .. only:: developer
+
+        Test: python -m hpat.runtests hpat.tests.test_series.TestSeries.test_series_isna
+        Test: python -m hpat.runtests hpat.tests.test_series.TestSeries.test_series_isna1
+
+    Parameters
+    -----------
+    self : :obj:`pandas.Series` object
+               input argument
+
+    Returns
+    -------
+    :obj:`pandas.Series`
+         returns :obj:`pandas.Series` object
```

Comment:
Can you please describe what this output series contains, e.g. 'that contains bool values indicating if corresponding element of self is NaN"